tests/test_lanes.py

```diff
@@ -978,34 +1021,35 @@ def test_constructor(self):
         # KnownOverviewFacetsWorkLists. They only show works that are
         # currently available.
         for i in available_now:
+            # Each lane is associated with the JackpotFacets we passed
+            # in.
             assert isinstance(i, KnownOverviewFacetsWorkList)
-            facets = i.facets
-            eq_(self._default_library, facets.library)
-            eq_(Facets.AVAILABLE_NOW, facets.availability)
+            internal_facets = i.facets
+            eq_(facets, internal_facets)
 
         # These worklists show ebooks and audiobooks from the two
         # collections associated with the default library.
         [default_ebooks, default_audio, overdrive_ebooks, overdrive_audio] = (
             available_now
         )
 
-        eq_("[Collection test] - License source {[Unknown]} - Medium {Book} - Availability {now} - Collection name {%s}" % self._default_collection.name, 
+        eq_("License source {[Unknown]} - Medium {Book} - Collection name {%s}" % self._default_collection.name,
```

Comment:
I realized that the logic that applied to removing "Availability {now}" also applied to removing "[Collection test]" -- you asked for the collection test so you know that's what you're getting.